CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# Contributing to json-graph-specification
+
+First off, thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to json-graph-specification, which are hosted in the [JSON Graph Organization](https://github.com/jsongraph) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[I don't want to read this whole thing, I just have a question!!!](#i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Pull Requests](#pull-requests)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [JSON Graph Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior.
+
+## I don't want to read this whole thing I just have a question!!!
+
+Feel free to [open an issue](https://github.com/jsongraph/json-graph-specification/issues/new) at any time.
+
+## What should I know before I get started?
+
+Before opening issues about topics that are already solved, please read the [readme file](https://github.com/jsongraph/json-graph-specification#readme) that is describing what this project is about and how it works. Also, have a look at the [project page](http://jsongraphformat.info/).
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report. Following these guidelines helps maintainers and the community understand your report, reproduce the behavior, and find related reports.
+
+Before creating bug reports, please check [the open issues](https://github.com/jsongraph/json-graph-specification/issues) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-good-bug-report).
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+#### How Do I Submit A (Good) Bug Report?
```

Comment:
We may want to describe and link out the [SSCCE](http://sscce.org/) concept.

Reply:
I added a sentence about this. Do you think this is enough, or should we explain more before linking out? I also like the [MCVE that Stackverflow is using](https://stackoverflow.com/help/mcve).

Reply:
My approach to this is if we start getting a lot of issues that are not created well, we'll address this then with more structure (I prefer Issue templates for this actually), but I don't believe in making things more complicated unless they require it.